src/Init/Prelude.lean

```diff
@@ -3342,17 +3370,22 @@ end
 section
 variable {ρ : Type u} {m : Type u → Type v}
 
-/-- `(← read) : ρ` gets the read-only state of a `ReaderT ρ`. -/
+/-- Retrieves the reader monad's local value. -/
```

Comment:
With the Nat docstrings, there was guidance to use, eg, / over Nat.div. Do you want to say something similar here, to use `MonadReader.read` over `ReaderT.read`? (In this case I don't know which one is actually preferred!)

Reply:
I'll add a comment to that effect. Thanks!